.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I think this feature request template would be a bit better if it was more related to APIs and not GUI app

Reply:
Do you have specific questions in mind?

Reply:
It's more clear now than it's showing an example related to one of our APIs